data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1011,84 +1011,127 @@ mission "Hai Reveal [B13] Freedom"
-			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a large blast door. Only a couple dozen operatives are critically or fatally wounded, but the pirates were very good at booby trapping things or utilizing dormant machinery to pin, crush, or disable operatives or their suits. A great many of them will be spending time in a hospital. The Unfettered fared comparatively better, having mostly acted as reserve and flanking forces; still, over a dozen of them will be taking their own tails home as trophies after a few groups of pirates used miner's cutting power-arm extensions to great effect.`
-			`	Having worked through the spaceport and down the access tunnel to the settlement, the vast blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side though, and has not been sealed very well against being reopened from this side. Before long it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`
+			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred operatives pour out wearing armor-plated exoskeletons. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`
+			`	The squads move room to room, neutralizing all opposition with ease until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the suits (and all other electronics nearby), leaving affected operatives helpless for ten seconds. Here some of General Choot'k's Unfettered show their strength, darting between operatives and bounding off walls with pulse rifles and long knives. They strike rapidly and withdraw as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`
+			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a blast door. Only a couple dozen operatives are critically or fatally wounded, but the pirates were very good at trapping the suits. The Unfettered fared comparatively better, having mostly acted as reserve and flanking forces; still, over a dozen of them will take their own tails home as trophies after a pirates used miner's cutting power-arm extensions to great effect.`
```

Comment:
```suggestion
			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a blast door. Only a couple dozen operatives are critically or fatally wounded, but the pirates were very good at trapping the suits. The Unfettered fared comparatively better, having mostly acted as reserve and flanking forces, and being evidently better trained and acting together, supporting those unfavorable positions and withdrawing when the time is right; still, over a dozen of them will take their own tails home as trophies after a pirates used miner's cutting power-arm extensions to great effect.`
```
Doesnt have to be this exactly but I find it weird they're described as being in reserve like this and their little casualties being only linked to that

Reply:
The number of Syndicate operatives that are "casualties" compared to Unfettered is also inconsistent with the other statement that only a few of them were injured, and most are trapped. I think NV lost track of her casualty rates.

Reply:
they were outnumbered 1/12 so you'd expect casualties